src/main/java/com/ably/kafka/connect/ChannelSinkTask.java

```diff
@@ -105,6 +91,10 @@ public void stop() {
         logger.info("Stopping Ably channel Sink task");
         ablyClient.stop();
```

Comment:
Since this is now using `AblyRest`, `stop()` does nothing

Reply:
Removed.